tasmota/xsns_54_ina226.ino

```diff
@@ -237,7 +238,7 @@ void Ina226Init()
 
     if (!I2cWrite16( addr, INA226_REG_CONFIG, INA226_CONFIG_RESET)){
 
-      AddLog_P2( LOG_LEVEL_DEBUG, "No INA226 at address: %02X", addr);
+      //AddLog_P2( LOG_LEVEL_DEBUG, "No INA226 at address: %02X", addr);
```

Comment:
This should be enabled . The user will see this only if weblog, syslog, mqttlog or serialog are set to LOG_LEVEL_DEBUG (3)

Reply:
Reverted.